SQLiteStudio3/coreSQLiteStudio/common/utils.cpp

```diff
@@ -679,50 +679,7 @@ int sum(const QList<int>& integers)
 QString getOsString()
 {
 #if defined(Q_OS_WIN)
-    QString os = "Windows";
-    switch (QSysInfo::WindowsVersion)
-    {
-        case QSysInfo::WV_XP:
-            os += " XP";
-            break;
-        case QSysInfo::WV_2003:
-            os += " 2003";
-            break;
-        case QSysInfo::WV_VISTA:
-            os += " Vista";
-            break;
-        case QSysInfo::WV_WINDOWS7:
-            os += " 7";
-            break;
-        case QSysInfo::WV_WINDOWS8:
-            os += " 8";
-            break;
-        case QSysInfo::WV_WINDOWS8_1:
-            os += " 8.1";
-            break;
-#if QT_VERSION >= 0x050500
-        case QSysInfo::WV_WINDOWS10:
-            os += " 10";
-        break;
-#endif
-        case QSysInfo::WV_32s:
-        case QSysInfo::WV_95:
-        case QSysInfo::WV_98:
-        case QSysInfo::WV_Me:
-        case QSysInfo::WV_DOS_based:
-        case QSysInfo::WV_NT:
-        case QSysInfo::WV_2000:
-        case QSysInfo::WV_NT_based:
-        case QSysInfo::WV_CE:
-        case QSysInfo::WV_CENET:
-        case QSysInfo::WV_CE_5:
-        case QSysInfo::WV_CE_6:
-        case QSysInfo::WV_CE_based:
-#if QT_VERSION >= 0x050500
-        case QSysInfo::WV_None:
-#endif
-            break;
-    }
+    QString os = QSysInfo::prettyProductName();
```

Comment:
Nice :)

Reply:
I am not sure it produces the same result; however, I also don't see this code being used anywhere

Reply:
Good catch. I scanned git history for that and it turns out that it used to be referenced from Bug Reporting dialog, back in the days, when reporting was done directly from build-in dialog. Later it was changed, so that the "Report Bug" menu entry opens up GitHub "New Issue" page in the browser and the dialog was removed. Therefore I believe this function can be removed as well. No need to maintain unused code.

Reply:
Ok, I have removed the function altogether.